### DISCUSSION TITLE
Allowing only 3 pinned items in issues seems too low for any serious project

### DISCUSSION THREAD

**Discussion by TinCanTech** [community/community#3148] [Discussions]
My project is tiny and yet I would still like 4 pinned items currently.

**Comment by evi-liu** [comment]
You can pin up to 4 discussions! We've limited to that number as we haven't heard that communities need more than 4.

**Comment by TinCanTech** [comment]
My project is tiny and yet I have **Four Issues** which I would specifically like to pin.

**Reply by evi-liu** [reply]
This space is currently reserved for feedback on **Discussions** and **GitHub for Mobile**, not **Issues** unfortunately. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).

**Reply by TinCanTech** [reply]
> This space is currently reserved for feedback on **Discussions** and **GitHub for Mobile**, not **Issues** unfortunately. 

My initial post clearly stated **Issues** as the point.

Therefore, your _answer_ is **not** an answer, it is M$ changing the subject in their favour.

*** VOTE TRUMP ***

Bury this left wing garbage.

**Comment by TinCanTech** [comment]
The4 _so-called_ Answer above is **not** an answer to the Original Question.

But due to Microshaft being in charge, I no longer get an option to remove the **Wrong** "Answer Badge".
